astropy/io/ascii/src/tokenizer.c

```diff
@@ -47,8 +47,8 @@ void delete_data(tokenizer_t *tokenizer)
     int i;
 
     if (tokenizer->output_cols)
-	for (i = 0; i < tokenizer->num_cols; ++i)
-	    free(tokenizer->output_cols[i]);
+    for (i = 0; i < tokenizer->num_cols; ++i)
+        free(tokenizer->output_cols[i]);
```

Comment:
The indentation level doesn't match the logic here.  I generally prefer explicit braces in C-like languages for this very reason, though I know that's not the style used elsewhere in this file.